Fix for pytest (python3).

myakove opened the pull request:
get_marker() is gone on pytest (py3).

myakove committed:
Fix for pytest (python3).

get_marker() is gone on pytest (py3).

myakove committed:
flake8: set max line to 120

myakove committed:
Remove tests for py3.4 and add py3.7

lukas-bednar commented on tox.ini:
```diff
@@ -1,8 +1,8 @@
 [tox]
-envlist = py27,py34,pep8
+envlist = py27,py37,pep8
```
I believe that it would be good to add matrix for pytest3.x & pytest4.x .
So we have compatibility ensured.

myakove committed:
Run tests on pytest 3.x and 4.x

Test metrix:
  py27 with pytest 3.6.1
  py27 with pytest 4.3.1
  py37 with pytest 3.6.1
  py37 with pytest 4.3.1

lukas-bednar reviewed (approved):
LGTM

lukas-bednar commented:
Thank you!

lukas-bednar commented:
@myakove since this repo is just a customization of [eanxgeek/pytest_marker_bugzilla](https://github.com/eanxgeek/pytest_marker_bugzilla). It would be great if we can integrate this improvement there as well. Would you be so kind  to do that ? If you don't have a time I will deal with that ,it is not problem.

myakove commented:
@lukas-bednar eanxgeek/pytest_marker_bugzilla is outdated and not maintained.
I think the right thing to do it to ask @eanxgeek to get us maintainer permissions or create our own package in pypy from our fork.

What do you think?

lukas-bednar commented:
@myakove I have maintainer's permissions. And it is not outdated, it just don't have our customizations on top of it.
